src/Ext/Script/Body.ManageAITriggers.cpp

```
@@ -0,0 +1,195 @@
+#include "Body.h"
+
+void ScriptExt::ManageTriggersFromList(TeamClass* pTeam, int idxAITriggerType = -1, bool isEnabled = false)
+{
+	if (!pTeam)
+		return;
```

This kind of `nullptr` check is really not needed. Generally speaking if the pointer of 'this' or 'current' object is null, you have a much bigger fish to fry than your code crashing (in fact, it is probably better to let it crash than have it obscure a potential issue).